runners/wikiConsole/templates/createFolder.asciidoc

```diff
@@ -0,0 +1,14 @@
+Create Folder
+=============
+
+
+Creating a new Folder
+~~~~~~~~~~~~~~~~~~~~~
+
+A folder can be created either from within an IDE like VSCode or Eclipse by simply right clicking in the file explorer and creating a new folder. Another approach is to use the Terminal. 
+
+On Linux or MacOS, you can use any file explorer like Finder or execute the command `mkdir -p <%= folderPath; %>`. The `-p` flag won't throw an error if the target folder already exists and allows the creation of parent directories.
+
+On Windows CMD, you can use the Windows Explorer to create folders or use the batch command `if not exist <%= folderPath; %> mkdir <%= folderPath; %>`.
+
+If you're using Windows Powershell, use the command `New-Item -ItemType Directory -Force -Path <%= folderPath; %>`.
```

Comment:
I think the right spelling is: PowerShell. Since it's a name, I think we should write it like this, similar to GitHub